src/main/java/org/folio/services/validator/FundAvailabilityHolderValidator.java

```diff
@@ -77,17 +76,21 @@ private MonetaryAmount calculateTotalExpendedAmount(List<InvoiceWorkflowDataHold
       .orElseGet(() -> Money.zero(currency));
   }
 
+  /**
+   * Method is following these formulas <br>
+   * afterApproveExpended [remaining amount we can expend] = expended - credited + (totalFunding * allowableExpenditure) <br>
+   * expended = awaitingPayment + expenditure
+   */
   private boolean isRemainingAmountExceed(Budget budget, MonetaryAmount totalExpendedAmount) {
-    // [remaining amount we can expend] = (totalFunding * allowableExpenditure) - expended
-    // where expended = awaitingPayment + expenditure
```

Comment:
regarding this changes the logic is not the same with comment, so updated comment based on the code.

Reply:
This code is very confusing, please create a `awaitingPayment` variable to separate it from expended and rename `totalExpendedAmount` to `additionalAmountToExpend`.